app/views/admin/edition_unpublishing/edit.html.erb

```diff
@@ -1,31 +1,39 @@
-<% edition = @unpublishing.edition
-   withdrawal_or_unpublishing = withdrawal_or_unpublishing(@unpublishing.edition)
-%>
+<% content_for :context, @unpublishing.edition.title %>
+<% content_for :page_title, "Edit #{withdrawal_or_unpublishing(@unpublishing.edition)} explaination" %>
```

Comment:
I think this is a typo (yeah I know it follows the design) - "explanation" not "explaination"